src/StaticFAQ/FAQCategoryList.js

```diff
@@ -17,7 +17,7 @@ import BaggageInfo from './FAQExtraInfo/BaggageInfo';
 import BoardingPassesInfo from './FAQExtraInfo/BoardingPassesInfo';
 import FAQArticle from './FAQArticle';
 import FAQCategory from './FAQCategory';
-import Breadcrumbs from './Breadcrumbs';
+import Breadcrumbs from './breadcrumbs/Breadcrumbs';
```

Comment:
Maybe use `index.js` here? :stuck_out_tongue_closed_eyes: I know we don't really like it, but this looks redundant. Normally I probably wouldn't notice this, but this is a refactoring PR.

Reply:
it's not possible due to relay, that's why we usually don't like `index.js` files

Reply:
ah, I didn't realize that! You're right. Thanks